spec.bs

```diff
@@ -2015,6 +2003,15 @@ To <dfn>convert to an AuctionAd sequence</dfn> given a [=list=]-or-null |ads|:
     1. Let |adIDL| be a new {{AuctionAd}}.
     1. [=map/Set=] |adIDL|["{{AuctionAd/renderURL}}"] to the [=URL serializer|serialization=] of
       |ad|'s [=interest group ad/render url=].
+    1. If |ad|'s [=interest group ad/selectable buyer and seller reporting IDs=] is not null:
+      1.[=map/Set=] |adIDL|["{{AuctionAd/selectableBuyerAndSellerReportingIds}}"] to
+        |ad|'s [=interest group ad/selectable buyer and seller reporting IDs=].
+      1. If |ad|'s [=interest group ad/buyer and seller reporting id=] is not null then
+        [=map/set=] |adIDL|["{{AuctionAd/buyerAndSellerReportingId}}"] to
+        |ad|'s [=interest group ad/buyer and seller reporting id=].
+      1. If |ad|'s [=interest group ad/buyer reporting id=] is not null then
+        [=map/set=] |adIDL|["{{AuctionAd/buyerReportingId}}"] to

```

Comment:
Here, it's [=map/set=], but on 2667, it's [=map/Set=], and in the lines above it's both. Is one correct, or is there a reason to alternate?

Reply:
There's no difference. It's just capitalized due to English habit (being the first character of a sentence).

Besides this, bikeshed can automatically handle a wide range of English conjugations and pluralizations (https://speced.github.io/bikeshed/#conjugating). For example, if you define the term "snap", you can link to it with "snapping" or "snapped"